roles/neutron-common/templates/etc/neutron/rootwrap.conf

```diff
@@ -12,9 +12,9 @@ filters_path=/etc/neutron/rootwrap.d,/usr/share/neutron/rootwrap
 # If not specified, defaults to system PATH environment variable.
 # These directories MUST all be only writeable by root !
 {% if openstack_install_method == "package" -%}
-exec_dirs=/sbin,/usr/sbin,/bin,/usr/bin,{{ 'neutron'|ursula_package_path(openstack_package_version) }}/bin
+exec_dirs=/sbin,/usr/sbin,/bin,/usr/bin,/usr/local/bin,{{ 'neutron'|ursula_package_path(openstack_package_version) }}/bin
 {% else -%}
-exec_dirs=/sbin,/usr/sbin,/bin,/usr/bin
+exec_dirs=/sbin,/usr/sbin,/bin,/usr/bin,/usr/local/bin
```

Comment:
Lets just make sure /usr/local/bin/ is in the exec path and try that. I don't think we actually need the package path in the exec_dirs.